uniffi/src/bindings/kotlin/templates/NamespaceLibraryTemplate.kt

```diff
@@ -22,9 +22,9 @@ internal interface _UniFFILib : Library {
     {% for func in ci.iter_ffi_function_definitions() -%}
         fun {{ func.name() }}(
         {%- for arg in func.arguments() %}
-            {{ arg.name() }}: {{ arg.type_()|type_c }}{% if loop.last %}{% else %},{% endif %}
+            {{ arg.name() }}: {{ arg.type_()|type_c }}{% if loop.last && !func.has_error() %}{% else %},{% endif %}
         {%- endfor %}
-        // TODO: When we implement error handling, there will be an out error param here.
+        {% match func.throws() %}{% when Some with (e) %}error: {{e}}.ByReference{% else %}{% endmatch %}
```

Comment:
The only functions I found that don't take errors are the free/alloc bytebuffers and free strings. And those are defined by macros in `ffi_support` :( If they did take errors, then this logic would be simpler (i.e we'd assume all functions take errors and get rid of the match)